scripts/Item.js

```diff
@@ -0,0 +1,41 @@
+export default class Item {
+    constructor(img, title, desc, price, id) {
+        this.img = img
+        this.title = title
+        this.desc = desc
+        this.price = price
+        this.id = id
+    }
+    getId() {
+        return this.id
+    }
+    getImg() {
+        return this.img
+    }
+    getTitle() {
+        return this.title
+    }
+    getDesc() {
+        return this.desc
+    }
+    getPrice() {
+        return this.price
+    }
+    setImg(img) {
+        this.img = img
+    }
+    setTitle(title) {
+        this.title = title
+    }
+    setDesc(desc) {
+        this.desc = desc
+    }
+    setPrice(price) {
+        this.price = price
+    }
+
+    makeHtmlItem() {
+        const st = "'" +this.img + "', " + this.title + ", " + this.desc + ", " + this.price + ", " + this.id
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
Plus, can't find the usage of `st`